x-pack/plugins/alerting/common/action_ref_prefix.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// NOTE: Changing this prefix will require a migration to update the prefix in all existing `rule` saved objects
+export const preconfiguredConnectorActionRefPrefix = 'preconfigured:';
+
+// NOTE: Changing this prefix will require a migration to update the prefix in all existing `rule` saved objects
+export const systemConnectorActionRefPrefix = 'system_action:';
```

Comment:
where do we have `system_action`?

Reply:
I just realized that we merged some code around system action already. So all good here